Sources/Settings/SettingsTabViewController.swift

```diff
@@ -95,11 +95,10 @@ final class SettingsTabViewController: NSViewController, SettingsStyleController
 			} else {
 				let settings: String
 				if #available(macOS 13, *) {
-					settings = Localization[.settings]
+                    settings = NSLocalizedString("settings", bundle: .module, comment: "Settings")
```

Comment:
Incorrect indentation.

Reply:
Got the diff. The original code use tab to indent while my new code use space to indent.

I'll change to use "tab" to follow the repo's general style.

<img width="621" alt="image" src="https://github.com/sindresorhus/Settings/assets/43724855/13e383a5-dddc-4ab2-a8ce-f40d3fc2700e">